client/src/components/header/SignUp.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import Modal from 'react-modal';
+import axios from 'axios';
+
+export default class SignUp extends Component {
+  constructor(props){
+
+    super(props);
+
+    this.state = {
+      username: null,
+      password: null,
+      email: null,
+      firstName: null,
+      lastName: null,
+      loggedIn: false,
+    }
+    
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSignup = this.handleSignup.bind(this);
+  }
+
+  handleChange(e){
+    this.setState(
+        {
+            [e.target.name]: e.target.value
+        }
+    )
+  }
+
+  handleSignup(credentials) {
+    return axios.post('/api/signup', credentials)
+      .then(() => {
+        this.props.sendLogin(this.state);
+        console.log('successs')
```

Comment:
I think this console.log can be removed or changed to be more descriptive if you think it's necessary